### ISSUE TITLE
More than 1 fingerprint in a YAML file

### ISSUE THREAD

**Issue by anshumanbh** [ztgrace/changeme#48]
Lets take Apache Tomcat as an example. 

There is Tomcat Manager and Tomcat Host Manager. They both have different endpoints and different fingerprints. For instance, the Basic Auth Realm for Tomcat Manager says "Tomcat Manager Application" and for Tomcat Host Manager, it says "Tomcat Host Manager Application". 

Tomcat Host Manager is exposed at the endpoint `/host-manager/html` whereas Tomcat Manager is exposed at endpoints like `/manager/html`, `/tomcat/manager/html`, `/manager/text`, `/manager/status`.

So, basically even though it is Apache Tomcat, it needs multiple sections of fingerprints. 

Right now, I just created two separate files for them, but it would be nice to be able to have multiple sections of fingerprints in a single YAML file.

**Comment by ztgrace** [comment]
@anshumanbh Does Tomcat Host Manager have the same credentials as Tomcat Manager?

**Comment by anshumanbh** [comment]
@ztgrace From my experiences, yes!

**Comment by ztgrace** [comment]
@anshumanbh Looking at this again, I thought for this specific case it was best to make a separate file so that the title would be accurate. Please take a look at https://github.com/ztgrace/changeme/blob/development/creds/http/general/apache_tomcat_host_manager.yml to see if I captured your description correctly and test it if you can. I also made sure to list you as the contributor.

I still like the idea of having multiple fingerprints and will implement that soon. An example might be if there's a web management interface and an API that would have the same default credentials. So I'm going to leave this issue open until that feature is closed.

**Comment by anshumanbh** [comment]
Looks good!
